dashboard/grid-template.html

```diff
@@ -117,7 +117,9 @@
     {% set row = loop.index0 %}
     <tr>
         <td><font face="verdana">{{ row_package.friendly_name }}</font>{% if results.has_issues(row_package) %}
-                <img class="warning-img" src="https://www.gstatic.com/images/icons/material/system/1x/warning_amber_18dp.png">
+                {% set issues = results.deprecated_deps[row_package.friendly_name][0] %}
+                <img class="warning-img" class="dialog" onclick="showIssues('{{ row_package.friendly_name|safe }}', '{{ issues|replace('\'', '\\\'')|safe }}')" src="https://www.gstatic.com/images/icons/material/system/1x/warning_amber_18dp.png">
```

Comment:
Can't jinja2 do this escaping for you? I'm not sure.

Reply:
Removed, jinja2 couldn't do the escaping for `'`, so I have to manually escape it here.